app/models/stripe_account.rb

```diff
@@ -77,3 +77,39 @@ def serialize_on_update(input)
     self.object
   end
 end
+
+
+class StripeAccountRequirements
```

Comment:
If there was a `StripeAccount` module that had the `Requirements` class? I'm not sure if it would be allowed to have a class and a module with the same name, though.